ethcore/src/client/client.rs

```diff
@@ -1449,27 +1453,26 @@ impl ImportBlock for Client {
 			return Err(EthcoreError::Block(BlockError::UnknownParent(unverified.parent_hash())));
 		}
 
+		let bytes = unverified.bytes.clone();
```

Comment:
we're cloning every block on import, not sure if
what are the advantages of that over making bytes part of `EthcoreError::Block`?

Reply:
It is still quite bad but if you see we are already cloning the block if the `!block_queue.is_empty()` which doesn't happen that often.

I have benchmarked it compared to master by importing 1_000_000 blocks from mainnet and I found no regression.

I still think it is better to have it in `EthcoreError::Block` but it will require alot of changes when I checked in the rest of the code base which is using `EthcoreError::Block`.

I can check one more time.